src/content/en/2021/css.md

```diff
@@ -892,10 +891,11 @@ We also saw a dramatic reduction at the top end of gradient complexity.  Last ye
   height=100
 ) }}
 
+{# TODO: (Authors) I think the figure caption can be expanded to say it goes from ugly yellow to blue to pink to orange... #}
```

Comment:
@meyerweb

Reply:
I kind of avoided describing it because I didn’t want to make aesthetic calls (e.g., what makes the yellow “ugly”?).  I can fill it out a bit in terms of straight description of color changes, but do we think that’s useful to the blind and/or search engines?

Reply:
yeah I was thinking for blind to describe it a bit more. I agree though its hard to describe it :)

Reply:
Done in the latest edits.